tools-of-quality.md

```diff
@@ -0,0 +1,112 @@
+
+## 👀 Transparency
+
+To ensure a wholesome and productive environment, we _must and will_ be fully transparent with the team.
+
+The customer is a part of the team. Therefore, if you "can't" say something to the customer, it does not exist for the team. We will not be forced to keep secrets from our own team members.
+
+In the same spirit, we will not do any double bookkeeping of issues, statuses, time etc. If this is "needed", we will have no part in it and the work will fall to whoever feels that this is "needed".
+
+## 💬 Language
+
+We strive to be as clear and concise as possible.
+
+For the sake of clarity we forgo the use of abbreviations unless these are widely understood (we will, for example, use `URL` instead of `Uniform Resource Locator`).
+
+We will write code, documentation, issues etc. in US English, as this is the language of the code we write, and makes it possible to include those not proficient in Norwegian as members of a team.
+
+If the customer demands that some of these things be done in Norwegian, an explicit choice must be made to satisfy that demand.
+
+Certain circumstances exist where mixing of languages is preferred, such as when a domain is only easily expressed in a given language.
+
+Regardless of language, we will strive to write and speak in a gramatically correct manner in order to avoid misunderstandings.
+
+There will be instances in which technical language, that is only understood by a person with sufficient proficiency, must be used.
+
+## 👩‍👩‍👦‍👦 Explicit Roles and Mandates
+
+We must know who is responsible for what. Therefore:
+
+- Mandates must be defined.
+- Roles must be defined.
+- Roles must have one or more mandates.
+- Members must be defined.
+- Members must have one or more roles.
+
+There must also exist a role with the mandate to define roles and mandates so that eventual missing or inconsistent roles and mandates can be (re)defined.
+
+Creuna has this already, including several defined roles and mandates, but the defined roles and mandates might not (but should) be sufficient for most projects.
+
+It's perfectly cromulent to not define extra roles or mandates, and also grow the team, mandates, and roles organically as development progresses.
+
+## 📝 Issue Tracking
+
+A way to track issues must exist.
+
+The issues must be visible to, at least, the project members.
+
+The status of an issue must be visible on the issue itself, and reflect the actual status of the issue in the real world.
+
+This means that the overall state of the system is visible and knowable. E-mail, calls and other private messages are therefore not an acceptable form of issue tracking.
+
+If an issue does not exist on the issue tracker, it does not (for the purposes of the project or system) exist.
+
+Work is derived from issues on the issue tracker by members suited to perform the work, not prescribed to specific members.
```

Comment:
"We strongly prefer pull based work" or something to that effect?

Reply:
Made a change to this.